platform/server/plugins/KibanaPluginValues.ts

```diff
@@ -23,7 +22,7 @@ export function createKibanaValuesForPlugin(
 ): KibanaPluginFeatures {
   return {
     logger: {
-      get: (...contextParts: string[]) => {
+      get: (...contextParts) => {
```

Comment:
Is there any reason why you don't want to use method shorthand syntax instead of anonymous arrow function here and in all definitions below?

```js
logger: {
  get(...) {
  } 
}
```
It seems anonymous arrow function grabs "global" context and `this` will be different from `this` that we would get with method shorthand?

Reply:
:100: ✅